tests/testthat/test-expect_lint.R

```diff
@@ -25,7 +25,7 @@ test_that("single check", {
 
   expect_error(expect_lint("a=1", c(message = msg, lineXXX = 1L), linter), "invalid field")
 
-  expect_failure(expect_lint("a=1", list(ranges = list(c(2L, 2L))), linter))
+  expect_success(expect_lint("a=1", list(ranges = list(c(2L, 2L))), linter))
```

Comment:
What was the original failure?

Reply:
> Error: check #1: ranges NULL did not match list(c(2L, 2L))

Reply:
So that seems like a way of documenting the bug we just fixed 😆 
Feel free to remove the test.

Reply:
Despite not really knowing the purpose of the test I'm still a bit loath to remove it :laughing:

I changed it to use a different linter that gives `NULL` for `ranges`